### PR TITLE
Cadvisor Configure Root Path

### PR DESCRIPTION
**Pull request by dashpole**
Cadvisor allows the RootPath to be configured.  The RootPath is used to determine which filesystem is the RootFs.

### CODE REVIEW

**Comment by dashpole** [comment]
@k8s-bot test this

**Comment by dashpole** [comment]
@k8s-bot test this

File: fs/fs.go [hunk 1]
```diff
@@ -115,7 +118,15 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	fsInfo.addDockerImagesLabel(context, mounts)
 
 	glog.Infof("Filesystem partitions: %+v", fsInfo.partitions)
-	fsInfo.addSystemRootLabel(mounts)
+	rootDevice, err := fsInfo.GetDirFsDevice(context.RootPath)
+	if err != nil {
+		return nil, fmt.Errorf("Error trying to get filesystem Device for rootPath %v: err: %v", context.RootPath, err)
```
Comment (timstclair) [inline fs/fs.go]: nit: error messages should start with a lowercase character (log messages start with Capital)

Same below.

Comment (dashpole) [inline fs/fs.go]: done

**Pull request merged**